- Cisco N2200 AC/DC Controllers

From: Shuotian Cheng <shuche@microsoft.com>

CONFIG_SENSORS_CISCO_N2200=m

- LM25066 and TPS40422 drivers
CONFIG_SENSORS_LM25066=m
CONFIG_SENSORS_TPS40422=m

- Unset SERIAL_8250_PNP
CONFIG_SERIAL_8250_PNP is not set
---
 debian/build/build_amd64_none_amd64/.config |    5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
index 718ae95..ab048d5 100644
--- a/debian/build/build_amd64_none_amd64/.config
+++ b/debian/build/build_amd64_none_amd64/.config
@@ -3469,7 +3469,7 @@ CONFIG_DEVMEM=y
 CONFIG_SERIAL_EARLYCON=y
 CONFIG_SERIAL_8250=y
 # CONFIG_SERIAL_8250_DEPRECATED_OPTIONS is not set
-CONFIG_SERIAL_8250_PNP=y
+# CONFIG_SERIAL_8250_PNP is not set
 CONFIG_SERIAL_8250_FINTEK=y
 CONFIG_SERIAL_8250_CONSOLE=y
 CONFIG_SERIAL_8250_DMA=y
@@ -3966,7 +3966,7 @@ CONFIG_SENSORS_LM25066=m
 # CONFIG_SENSORS_MAX34440 is not set
 CONFIG_SENSORS_DNI_DPS460=m
 # CONFIG_SENSORS_MAX8688 is not set
-# CONFIG_SENSORS_TPS40422 is not set
+CONFIG_SENSORS_TPS40422=m
 CONFIG_SENSORS_TPS53679=m
 CONFIG_SENSORS_UCD9000=m
 CONFIG_SENSORS_UCD9200=m
@@ -3974,6 +3974,7 @@ CONFIG_SENSORS_XDPE122=m
 CONFIG_SENSORS_MP2975=m
 # CONFIG_SENSORS_ZL6100 is not set
 CONFIG_SENSORS_DPS1900=m
+CONFIG_SENSORS_CISCO_N2200=m
 # CONFIG_SENSORS_SHT15 is not set
 CONFIG_SENSORS_SHT21=m
 # CONFIG_SENSORS_SHT3x is not set
